src/bp/core/services/ghost/service.ts

```diff
@@ -48,6 +48,7 @@ interface ScopedGhostOptions {
 }
 
 const MAX_GHOST_FILE_SIZE = process.core_env.BP_BPFS_MAX_FILE_SIZE || '100mb'
+const BP_BPFS_UPLOAD_CONCURRENCY = +(process.core_env.BP_BPFS_UPLOAD_CONCURRENCY || 50)
```

Comment:
Setting the value of `process.core_env.BP_BPFS_UPLOAD_CONCURRENCY` to something like `'abc'` would result in the variable being `NaN`. I think you can do something like this instead so that anything that isn't a number would fallback to 50:

```ts
const BP_BPFS_UPLOAD_CONCURRENCY = (parseInt(process.core_env.BP_BPFS_UPLOAD_CONCURRENCY, 10) || 50)
```

Reply:
`+(process.core_env.BP_BPFS_UPLOAD_CONCURRENCY as number) || 50 `

this worked a little better with the current number type

Reply:
I personally find it a little bit hackish to use the '+' operator to cast something into a number, but I think it has the same effect as parseInt in this case. So, it should be okay!